http/vibe/http/proxy.d

```diff
@@ -185,38 +200,57 @@ HTTPServerRequestDelegateS reverseProxyRequest(HTTPReverseProxySettings settings
 			else cres.bodyReader.pipe(res.bodyWriter);
 		}
 
-		try requestHTTP(rurl, &setupClientRequest, &handleClientResponse);
+		try requestHTTP(url, &setupClientRequest, &handleClientResponse);
 		catch (Exception e) {
 			throw new HTTPStatusException(HTTPStatus.badGateway, "Connection to upstream server failed: "~e.msg);
 		}
 	}
 
 	return &handleRequest;
 }
+
+/**
+	Returns a HTTP request handler that acts as a forward proxy
+*/
+HTTPServerRequestDelegateS forwardProxyRequest(HTTPProxySettings settings)
```

Comment:
I'd probably drop this, since the `settings.reverseProxyMode` becomes kind of redundant. In the long run, `reverseProxyRequest` could also be deprecated. Setting up proxies is also uncommon enough that  convenience functions don't really add much value in practice.